pkgs/top-level/all-packages.nix

```diff
@@ -9782,8 +9782,6 @@ with pkgs;
 
   trash-cli = callPackage ../tools/misc/trash-cli { };
 
-  trebleshot = libsForQt5.callPackage ../applications/networking/trebleshot { };
```

Comment:
Please add an entry to top-level/aliases.nix

Reply:
Done; did I do it right?

Reply:
Looks good!